### PR TITLE
HTML content and css styling on landing page

### PR DESCRIPTION
**Pull request by oscarV42**
Added a simple Landing page layout that can be improved.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        <a href="#" class="brand-logo">Trail Finder</a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+          <li><a href="#">About Us</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><a href="#">Search</a></li>
+        </ul>
+      </div>
+    </nav>
+    <div class="container" id="welcome">
+      <br /><br />
```
Comment (adam-niggebrugge) [inline index.html:27]: Trey says that <br> are too old school, we should use padding/margin in the finished project

**Review by NBK-01 — APPROVED** [review]
Looks good ! Could use some improvement in terms of color accessibility.

File: index.html (hunk 1 above)
Comment (adam-niggebrugge) [reply index.html:27]: `<br/>` was the tag i was comment

**Review by adam-niggebrugge — APPROVED** [review]
Great start. A few changes to avoid using the `<br>` tag when finializing

**Pull request merged**